cms/models/Item.js

```diff
@@ -150,25 +144,16 @@ const createWeeklyIfNeeded = async () => {
 Item.schema.post("save", async function(doc, next) {
 	await createWeeklyIfNeeded();
 
-	if(!shouldPostLambda){
-		shouldPostLambda = !this.isInitial;
-		next();
-	}	
-
 	logger.info("Post save, sending request...", doc);
 
-	var item = "";
-
-	await keystone.list("Item").model.findById(doc._id)
+	var item = await keystone.list("Item").model.findById(doc._id)
 	.populate("source")
 	.populate("evidenceType")
-	.then((source) => {
-		item = source;
-	})
+	.exec()
 	.catch((err) => {
 		logger.error("An error occurred finding source: ", err);
 		next(new Error(`An error occurred finding source: ${err}`));
```

Comment:
Probably make sense to return next here to stop executing - if we fail to retrieve the full item, then we shouldn't then try and post to the lambda